AlgebraInLean/Chapter2/Sheet3.lean

```diff
@@ -0,0 +1,232 @@
+import «AlgebraInLean».Basic
+import Mathlib.Tactic
+
+namespace Defs
+
+namespace Morphisms
+
+namespace Sheet3
+
+-- Injectivity, Surjectivity, Bijectivity
+  def Injective (f : α → β) : Prop := ∀ (x y : α), f x = f y → x = y
```

Comment:
You might want to pivot to using Mathlib's definitions: `Function.Injective`, etc. after they're introduced and discussed in Sheet 0.

Reply:
I believe I was told before starting this Interlude sheet that the goal was to stick with our 'proprietary' definitions of injectivity, subjectivity, etc. but we can discuss if people have been doing otherwise?